00new_packages.autobuild

```diff
@@ -13,5 +13,5 @@ end
 
 cmake_package 'gui/rock_webapp'
 ruby_package 'tools/rest_api' do |pkg|
-    Autoproj.env_set 'ROCK_WEBAPP', File.join(pkg.prefix, 'share', 'rest_api')
+    Autoproj.env_set 'ROCK_WEBAPP', File.join(Autoproj.build_dir, 'share', 'rest_api')
```

Comment:
> Why should an env var not refer to the build dir ?

Because it assumes that all the package prefixes are the main autoproj build directory. Which is not true if you use autoproj's extremely useful `separate_prefixes` feature.

For a more documented use-case, one that would create a pure ruby rock-webapp plugin can't use it.

In any case, the proper fix would be to have ROCK_WEBAPP a PATH instead of a single-path envvar. In the meantime, it could be set by gui/rock_webapp to wherever it is installed, instead of assuming a global install location.